input/pagecontent/index.md

```diff
@@ -1,48 +1,45 @@
-If an implementation is AU Core conformant this allows a level of core capability to be assumed when using.
+For an implementation guide that has basic constraints to support many uses it is recommended that the AU Core implementation guide is used.  AU Core uses AU Base, where available, as the basis for profiles that define the FHIR resources to be supported, and the elements, extensions, vocabularies, and value sets that SHALL be present are identified, and how they are used is defined.
 
 
+### Dependencies
+
+{% include dependency-table.xhtml %}
+
 ### How to Read this Guide
 
 This guide is divided into several pages which are listed at the top of each page in the menu bar.
 
 - [Home](index.html): This page provides the introduction and scope for this guide.
-- Guidance:
-  - [Guidance](guidance.html): This page provides guidance on using the profiles defined in this guide.
+- [Guidance](guidance.html):  These pages list the guidance for this guide.
+  - [General Guidance](generalguidance.html): This page provides guidance on using the profiles defined in this guide.
   - [Relationship with other IGs](relationship.html): This page provides guidance on the relationship between AU Base and other implementation guides.
   - [Future of AU Base](future.html): This page outlines the yearly update cycle and approach to maturing AU Base.
 - [FHIR Artefacts](artifacts.html): These pages provide detailed descriptions and formal definitions for all the FHIR artefacts defined in this guide.
-  - [Profiles and Extensions](profiles-and-extensions.html): This set of pages describes the profiles and extensions that are defined in this guide to represent Australian local concepts using FHIR. Each profile page includes a narrative description, guidance, and formal definition. Although the guidance typically focuses on the profiled elements and seeks to provide a ‘how-to’ guide when representing concepts, it may also may focus on un-profiled elements to aid with implementation.
-  - [Search Parameters](search-parameters.html): This set of pages lists the search parameters defined in this guide for use in AU operations.
-  - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
+  - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to represent Australian local concepts using FHIR. Each profile page includes a narrative description, guidance, and formal definition. Although the guidance typically focuses on the profiled elements and seeks to provide a ‘how-to’ guide when representing concepts, it may also may focus on un-profiled elements to aid with implementation.
+  - [Search Parameters](search-parameters.html): This page lists the search parameters defined in this guide for use in AU operations.
+  - [Terminology](terminology.html): This page lists the value sets and code systems defined in this guide.
```

Comment:
In the terminology line replace "defined" with "localised"

Reply:
Fixed, thanks.